frontend/src/component/project/Project/ProjectSettings/ProjectApiAccess/CreateProjectApiTokenForm.tsx

```diff
@@ -49,6 +50,7 @@ export const CreateProjectApiTokenForm = () => {
         useProjectApiTokensApi();
     const { refetch: refetchProjectTokens } = useProjectApiTokens(project);
     const { trackEvent } = usePlausibleTracker();
+    const { environments } = useProjectEnvironments(project);
```

Comment:
Will this only be enabled environments for the project?

Reply:
Yes, though the intention as I understand would be to use `projectVisible` instead of `enabled` so that what we have in Environments project settings
![Screenshot 2023-03-23 at 21 12 42](https://user-images.githubusercontent.com/104830839/227326123-9c1c4b69-0ba7-4224-84d6-cf0652294ce5.png)

is available in the dropdown

![Screenshot 2023-03-23 at 21 21 30](https://user-images.githubusercontent.com/104830839/227326176-1fe717d2-7d6a-4839-b1b8-ea21c589effa.png)

Reply:
yes exactly! thank you @andreas-unleash